tests/helm/testdata/goldenfile/traces-gateway-deployment/basic.output.yaml

```diff
@@ -48,11 +48,6 @@ spec:
                 secretKeyRef:
                   name: sumologic
                   key: endpoint-traces
-            - name: SUMO_ENDPOINT_DEFAULT_OTLP_TRACES_SOURCE
```

Comment:
is this removed from the template because the otlp source is disabled by default?

Reply:
Yes.